### PR TITLE
revise about page

### PR DESCRIPTION
**Pull request by kevinb9n**
Tried to turn the "about page" into more specifically an "about us" page. In the process I didn't quite feel like nuking the "what are we trying to do" text. It does seem like it belongs somewhere else, but I didn't want to try to solve that problem yet.

### CODE REVIEW

File: docs/src/pages/about.md [hunk 1]
```diff
@@ -55,8 +23,28 @@ People from the following projects and organizations are collaborating on this p
 | [SpotBugs](http://spotbugs.rtfd.io/)                    | [SpotBugs Team](https://github.com/spotbugs/) |
 | (various)                                               | [Square](https://squareup.com)                |
 
+## What are we trying to do?
+
+Java static analysis tools, such as Error Prone and those built into IDEs, have been very successful
+at enabling automated refactorings and bug prevention.
+
+Many of their most beneficial analyses require certain Java annotations (such as `@Nullable` or
+`@Immutable`) to be present in the source or class file. Yet for the going-on-two-decades since
```
Comment (scolsen) [inline docs/src/pages/about.md:32]: nit: "source or class file being analyzed" or pluralize it: "in source or class files"

Comment (kevinb9n) [reply docs/src/pages/about.md]: done

File: docs/src/pages/about.md [hunk 2]
```diff
@@ -55,8 +23,29 @@ People from the following projects and organizations are collaborating on this p
...
+at enabling automated refactorings and bug prevention.
+
+Many of their most beneficial analyses require certain Java annotations (such as `@Nullable` or
+`@Immutable`) to be present in the source or class file being analyzed. Yet for the 
+going-on-two-decades since Java annotations were introduced, there's been no serious and 
+successful effort to provide a single well-specified standard for these annotations with 
+industry consensus. Instead, each tool has generally provided its own, and each tool interprets 
+the meanings of the annotations differently.
+
+We think the lack of such an artifact has been one of the factors constraining the prevalance and
```
Comment (netdpb) [inline docs/src/pages/about.md:38]:
```suggestion
We think the lack of such a standard has been one of the factors constraining the prevalance and 
```

You haven't said the word "artifact" yet.

**Review by netdpb — APPROVED** [review]

File: docs/src/pages/about.md (hunk 2 above)
Comment (kevinb9n) [reply docs/src/pages/about.md:38]: fixed in the other direction.

**Pull request merged**